steely/plugins/b.py

```diff
@@ -0,0 +1,31 @@
+COMMAND = '.b'
+__author__ = ('byxor',)
+
+
+def main(bot, author_id, message, thread_id, thread_type, **kwargs):
+    last_message = lambda: bot.fetchThreadMessages(thread_id=thread_id, limit=2)[1]
+    send = lambda message: bot.sendMessage(message, thread_id=thread_id, thread_type=thread_type)
```

Comment:
hey paul
why are these lambdas?

Reply:
I like doing it because it separates the plugin logic from the nitty gritty bot parameters.

`send(encode(last_message()))` looks really simple to me.

It's not necessary.